lib/jormungandr/src/Cardano/Wallet/Jormungandr/Binary.hs

```diff
@@ -388,12 +383,14 @@ getMilli = label "getMilli" $ Milli <$> getWord64be
 getLeaderId :: Get LeaderId
 getLeaderId = label "getLeaderId" $ LeaderId <$> getByteString 32
 
-getLinearFee :: Get LinearFee
-getLinearFee = label "getLinearFee" $ do
-    const' <- Quantity <$> getWord64be
-    perByte <- Quantity <$> getWord64be
-    perCert <- Quantity <$> getWord64be
-    return $ LinearFee const' perByte perCert
+getLinearFee :: Get FeePolicy
+getLinearFee = label "getFeePolicy" $ do
+    a <- getWord64be
+    b <- getWord64be
```

Comment:
I think it's a shame to lose the documentation of what the first and second number represent by naming them `a` and `b` 🤷‍♂

Reply:
Well, they are coefficient in a linear equation, which are usually called `a` and `b`, that's why. We could be more verbose though I agree.